app/views/podcasts/show.rss.builder

```diff
@@ -159,10 +159,10 @@ xml.rss "xmlns:atom" => "http://www.w3.org/2005/Atom",
               url: ep.enclosure_url(@feed))
           end
 
-          if ep.transcript?
+          if ep.ready_transcript
```

Comment:
ok, and then this conditional makes more sense now; not only does the transcript have to be present, but the episode status has to be complete (and therefore, valid transcript is present?). and within the conditional, we're only using what is available because of the conditional; we know that `ready_transcript` is not nil because we've already entered the conditional for it, so we should not encounter nil method errors while inside the conditional. so this solves the core issue of the bug.

Reply:
> but the episode status has to be complete

Close - the _transcript_ status has to be complete.  (But you probably meant that anyways).